common/src/main/java/com/wynntils/wc/custom/item/properties/SkillPointProperty.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.wc.custom.item.properties;
+
+import com.wynntils.mc.utils.ComponentUtils;
+import com.wynntils.mc.utils.ItemUtils;
+import com.wynntils.utils.objects.CustomColor;
+import com.wynntils.wc.custom.item.WynnItemStack;
+import net.minecraft.ChatFormatting;
+
+public class SkillPointProperty extends CustomStackCountProperty {
+    public SkillPointProperty(WynnItemStack item) {
+        super(item);
+
+        char colorCode = ComponentUtils.getCoded(item.getHoverName()).charAt(16);
+        // Current skill point amount is always on line 4 i.e. index 3.
+        String pointsLine = ItemUtils.getLore(item).get(3);
+        String points = pointsLine.substring(6, pointsLine.indexOf('p') - 1);
```

Comment:
Am I paranoid if I would rather put this whole block in a try catch?

Reply:
An exception there won't crash the client and will just make the item invisible. 
But if I add safeguards I'd have to check whether the name has 17 characters or more, line 4 exists in the lore and that the points line is longer than 6 chars and contains a p after index 6. 

If we expect the lore to change I would just iterate over all the lore lines with a regex. A performance hit I was trying to avoid.

Reply:
Let's just skip it then, it will be fine.

Reply:
I would like a second opinion on this choice though. @P0keDev or @magicus any thoughts?

Reply:
I'm not really fond of "crashing instead of checking" as a programming style, even if we think that this will not affect anything. At the very least, the stack trace will clutter up the log, maybe hiding other, real issues. 

I think you can surround it with a `try { ... } catch (NPE | IOOB) { // ignore }` and just be happy about it?